egs/wsj/s5/steps/nnet3/xconfig_to_configs.py

```diff
@@ -219,17 +225,19 @@ def write_config_files(config_dir, all_layers):
             raise
 
 
-def add_back_compatibility_info(config_dir, nnet_edits=None):
+def add_back_compatibility_info(config_dir, existing_model=None,
```

Comment:
This function is differently named in kaldi_52.  Please make this PR against kaldi_52 as it will save me hassle in merging.